src/charactersheet/models/common/player_types.js

```diff
@@ -10,7 +10,7 @@ export var PlayerTypes = {
     dm: {
         key: 'dm',
         visibleTabs: [
-            'encounter', 'overview', 'dmscreen', 'party', 'chat', 'notes'
+            'encounter', 'overview', 'dmscreen', 'party', 'chat', 'notes', 'exhibit'
```

Comment:
Should we remove `chat` for this PR? I know we probably just dont let the tab render, but it might be good to remove from here, too?